test/java/seedu/addressbook/storage/StorageFileTest.java

```diff
@@ -60,6 +64,15 @@ public void load_validFormat() throws Exception {
     }
 
     @Test
+    public void load_nonExistantFile_returnsEmptyAddressBook() throws Exception {
+        AddressBook actualAB = getStorage(NON_EXISTANT_FILE_NAME).load();
+        AddressBook expectedAB = new AddressBook();
+        
+        assertEquals(actualAB, expectedAB);
+        assertFileDoesNotExist(TEST_DATA_FOLDER + "/" + NON_EXISTANT_FILE_NAME);
```

Comment:
Add a comment to explain that we are verifying the file is not created during loading. Otherwise the reader might wonder why this verification is done at this point.